source/simulator/assembly.cc

```diff
@@ -999,6 +1034,11 @@ namespace aspect
                                   scratch.material_model_outputs.viscosities[q]
                                   :
                                   std::numeric_limits<double>::quiet_NaN());
+              const double eta_two_thirds = (rebuild_stokes_matrix
+                                             ?
+                                             scratch.material_model_outputs.viscosities[q] * 2.0 / 3.0
+                                             :
+                                             0.0);
```

Comment:
This value (the 0.0) should not be used, right? If so, can you set it to `std::numeric_limits<double>::quiet_NaN()` like in the statement above? This keeps us on the safe side.